core/src/database/postgres/sql_type_wrapper.rs

```diff
@@ -1293,32 +1286,45 @@ fn map_dynamic_uint_to_ethereum_sql_type_wrapper(
 fn map_log_token_to_ethereum_wrapper(
     abi_input: &ABIInput,
     token: &DynSolValue,
-) -> EthereumSqlTypeWrapper {
+) -> Vec<EthereumSqlTypeWrapper> {
```

Comment:
It's unlikely to be the bottleneck for this kind of an application but putting everything in a vec does mean more heap allocations as far as I understand, just something to be mindful of, but i see the need for a list type for the tuple so seems fine enough to me given the functionality gain